common/djangoapps/common_initialization/apps.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
Our goal at one point (which still stands AFAIK) was to not add anything else under edx-platform/common. Instead, all new djangoapps would go into openedx.core.djangoapps.

Reply:
The reason why: common/djangoapps is a legacy tangle of dependencies that we've attempted to unravel as we've moved things out into openedx, which is supposed to only contain "clean"(-er) code.

Reply:
Awesome!  That answers an ongoing question I've had since starting here over a year ago.